styles.css

```diff
@@ -18,6 +18,33 @@ p {
     background-color: #1BAB49;
 }
 
+.ingredient {
+    border-radius: 15px;
+    box-shadow: 0 3px 15px rgba(0,0,0,0.2);
+    background-color: #1BAB49;
+    color: #FFFFFF;
+    float: left;
+    margin: 5px;
```

Comment:
Suggestion - change this to just margin-top. Helps bunch the ingredients together more.